java/client/src/main/java/com/youtube/vitess/client/cursor/Row.java

```diff
@@ -117,7 +133,7 @@ public UnsignedLong getULong(String columnLabel) throws SQLException {
   }
 
   public UnsignedLong getULong(int columnIndex) throws SQLException {
-    return (UnsignedLong) getAndCheckType(columnIndex, UnsignedLong.class);
+    return getObject(columnIndex, UnsignedLong.class);
```

Comment:
I thought about that, but in my opinion the least ambiguous convention is "return null for any getter where it is possible to return null". This seems to be the convention followed by JDBC, which returns null for `BigDecimal getBigDecimal()`.